packages/mutator-specification/src/ArrayLiteralMutatorSpec.ts

```diff
@@ -4,6 +4,10 @@ import ExpectMutation from './ExpectMutation';
 
 export default function ArrayLiteralMutatorSpec(name: string, expectMutation: ExpectMutation) {
   describe('ArrayLiteralMutator', () => {
+    beforeEach(() => {
+      console.warn('DEPRECATED: please migrate from the ArrayLiteralMutatorSpec to the ArrayDeclarationMutatorSpec');
```

Comment:
Is this really needed? I mean, is the `mutator-specification` also part of our public api?

Reply:
Let's leave it for now